docs/guides/collecting-openshift-logs-with-victoria-logs/README.md

```diff
@@ -0,0 +1,183 @@
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: vl-collect-audit
+subjects:
+  - kind: ServiceAccount
+    name: victorialogs
+    namespace: openshift-logging
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: collect-audit-logs
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: vl-collect-application
+subjects:
+  - kind: ServiceAccount
+    name: victorialogs
+    namespace: openshift-logging
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: collect-application-logs
+```
+
+Alternatively, you can use OpenShift console to create the service account and cluster role binding. 
+Navigate to `ServiceAccounts` and click on `Create Service Account`. Fill in the name `victorialogs` and namespace `openshift-logging` and click on `Create`. 
+Then, navigate to `RoleBindings` and create a binding for each `ClusterRole` for subject `victorialogs` in `openshift-logging` namespace.
+
+
+## Install Victoria Logs
+
+Add Victoria Metrics Helm [repository](https://github.com/VictoriaMetrics/helm-charts):
+```bash
+helm repo add vm https://victoriametrics.github.io/helm-charts/
+```
+
+Minimal configuration for VictoriaLogs running in OpenShift:
+```yaml
+securityContext:
+  enabled: true
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+      - ALL
+  readOnlyRootFilesystem: true
+
+podSecurityContext:
+  enabled: true
+  fsGroup: 1000660000
+  runAsNonRoot: true
+  runAsUser: 1000660000
+```
+Save the configuration to `vl.yaml`.
+
+
+Create namespace for VictoriaLogs:
+```bash
+kubectl create namespace vl
+```
+
+Install VictoriaLogs with the following command:
+```bash
+helm upgrade --namespace vl --install vl vm/victoria-logs-single -f vl.yaml
+```
+
+## Configure logs forwarding
+
+Cluster logging operator uses `ClusterLogForwarder` resource for logs forwarding configuration.
+
+Here is an example configuration to forward all cluster logs to VictoriaLogs:
+```yaml
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: logging
+  namespace: openshift-logging
+spec:
+  managementState: Managed
+  outputs:
+    - elasticsearch:
+        index: logs
+        url: "http://vl-victoria-logs-single-server.vl.svc.cluster.local:9428/insert/elasticsearch/_bulk?_stream_fields=log_type,hostname,stream,kubernetes.pod_name,kubernetes.container_name,kubernetes.pod_namespace&_time_field=@timestamp&_msg_field=message,msg,_msg,log.msg,log.message,log&fake_field=1"
+        version: 8
+        tuning:
+          compression: gzip
+      name: victorialogs
+      type: elasticsearch
+    - elasticsearch:
+        index: logs
+        url: "http://vl-victoria-logs-single-server.vl.svc.cluster.local:9428/insert/elasticsearch/_bulk?_stream_fields=log_type,hostname,annotations.authorization.k8s.io/decision,hostname,verb&_time_field=@timestamp&_msg_field=annotations.authorization.k8s.io/reason&fake_field=1"
```

Comment:
There are lot more audit fields, perhaps the docs should specify that this URL should be updated if some data is missing

Reply:
@vrutkovs These fields are only used to identify [the stream](https://docs.victoriametrics.com/victorialogs/keyconcepts/#stream-fields) for ingested log entries. All log fields which are sent will be saved and available for searching after the ingestion.

For example, on a playground - https://play-vmlogs.victoriametrics.com
Stream is identified by a few fields like `container_name`, `pod_name`, `namespace` and `stream`
But other fields are still available for search and visualized in expanded view
![image](https://github.com/user-attachments/assets/3ffca39f-0599-4470-9c24-8cf554b68915)

Reply:
oooh, gotcha